src/controllers/CommentReactionController.js

```diff
@@ -0,0 +1,109 @@
+import models from '../db/models';
+import Response from '../helpers/response';
+import LikeUnlikeComment from '../db/service/commentReactionSummary';
+
+const { Comment, CommentReaction } = models;
+/** CommentReaction Controller */
+class CommentReactionController {
+  /**
+   * @static
+   * @param {Object} req
+   * @param {object} res
+   * @param {object} done
+   * @return {object} comment
+   */
+  static async likeComment(req, res, done) {
+    try {
+      const comment = await Comment.findById(req.params.commentId);
+      if (!comment) {
+        const response = new Response(
+          'Not Found',
+          404,
+          'This comment does not exist',
+        );
+        return res.status(response.code).json(response);
+      }
+      LikeUnlikeComment.like(req.params.commentId);
+      CommentReaction.findOrCreate({
+        where: {
+          $and: [{ commentId: req.params.commentId }, { userId: req.verifyUser.id }]
+        },
+        defaults: {
```

Comment:
Why do you need this? Looks like repetition to me.

Reply:
the defaults? I am passing in the userId and commentId into the database, I don't think there is any repetition there

Reply:
What is the default doing exactly?

Reply:
I'm specifying the default values to be inserted into the database fields that's the `userId` and the `commentId`, those are the values that will be inserted whether it is updating or creating